src/common/Config/ConfigMgr.h

```diff
@@ -24,6 +24,8 @@ namespace Sapphire::Common
       template< class T >
       T getValue( const std::string& section, const std::string& name, T defaultValue = T() )
       {
+        #pragma warning( push )
```

Comment:
this pragma warning may only signal for MSVC (visual studio) compilers, not gcc or clang